solidity/dashboard/src/components/Routing.jsx

```diff
@@ -84,4 +84,31 @@ const CustomRoute = ({
   )
 }
 
+export const renderExplorerModePage = (PageComponent, index) => {
+  const finalPath = "/:address" + PageComponent.route.path
```

Comment:
I am not convinced about this solution. I think we should not modify routes on render. Maybe we can use query params in the url? Eg. `https://dashboard.keep.network/delegations/granted?address=0x0`. Then we can detect query param in the `App` component and use `Explorer` connector. Thoughts? 🤔

Reply:
Hmmm I think we are not modifying the routes on render here. I kept the `renderPage` method untouched and added another one (`renderExploreModePage`) to create routes for all pages with the address at the beginning. So for example for Liquidity Page we will have two routes:

/liquidity
/:address/liquidity

and this applies for all the page components.

But i agree with the second part of your comment - we can detect query param in the `App` component and use `Explorer` connector there 👍

Reply:
ahh right... I misunderstood 👍